system/Database/MySQLi/Connection.php

```diff
@@ -487,7 +487,10 @@ protected function _foreignKeyData(string $table): array
                         tc.TABLE_NAME,
                         kcu.COLUMN_NAME,
                         rc.REFERENCED_TABLE_NAME,
```

Comment:
Added Delete and Update Rules and Match option to data returned. This was done to all DBMS